.github/workflows/pr-build.yaml

```diff
@@ -0,0 +1,64 @@
+name: PR Build and Test
+
+on:
+  pull_request
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    name: Node.JS ${{ matrix.node-version }}
+    strategy:
+      matrix:
+        node-version: [12.x, 14.x]
+        axios-version: [v0.20.0-0, v0.21.1]
```

Comment:
It looks like a lot of Axios versions got dropped. Could you add those back so that we don't regress on older versions?

Reply:
The reason for that it the Axios library introduces breaking changes way too often and I don't want to regress people who don't upgrade (who can blame them)

Reply:
Initially I removed them because of issues with Yarn.  I can add them back.

Reply:
Done.